opentracing-spring-web-itest/mvc/src/test/java/io/opentracing/contrib/spring/web/interceptor/itest/mvc/MVCJettyITest.java

```diff
@@ -0,0 +1,197 @@
+        assertLogEvents(span.logEntries(), Collections.<String>emptyList());
+    }
+
+
+    @Test
+    public void testNoURLMapping() {
+        {
+            getRestTemplate().getForEntity("/nouUrlMapping", String.class);
+            Awaitility.await().until(reportedSpansSize(), IsEqual.equalTo(1));
+        }
+        List<MockSpan> mockSpans = TracingBeansConfiguration.mockTracer.finishedSpans();
+        Assert.assertEquals(1, mockSpans.size());
+        assertOnErrors(mockSpans);
+
+        MockSpan span = mockSpans.get(0);
+        Assert.assertEquals("GET", span.operationName());
+        Assert.assertEquals(404, span.tags().get(Tags.HTTP_STATUS.getKey()));
+
+        assertLogEvents(span.logEntries(), Collections.<String>emptyList());
+    }
+
+    @Test
+    public void testControllerMappedException() throws Exception {
+        {
+            getRestTemplate().getForEntity("/mappedException", String.class);
+            Awaitility.await().until(reportedSpansSize(), IsEqual.equalTo(1));
+        }
+        List<MockSpan> mockSpans = TracingBeansConfiguration.mockTracer.finishedSpans();
+        Assert.assertEquals(1, mockSpans.size());
+        assertOnErrors(mockSpans);
+
+        MockSpan span = mockSpans.get(0);
+        Assert.assertEquals("GET", span.operationName());
+
+        Assert.assertEquals(5, span.tags().size());
+        Assert.assertEquals(Tags.SPAN_KIND_SERVER, span.tags().get(Tags.SPAN_KIND.getKey()));
+        Assert.assertEquals("GET", span.tags().get(Tags.HTTP_METHOD.getKey()));
+        Assert.assertEquals(getUrl("/mappedException"), span.tags().get(Tags.HTTP_URL.getKey()));
+        Assert.assertEquals(409, span.tags().get(Tags.HTTP_STATUS.getKey()));
+        Assert.assertNotNull(span.tags().get(Tags.COMPONENT.getKey()));
+
+        assertLogEvents(span.logEntries(), Arrays.asList("preHandle", "afterCompletion"));
+    }
+
+    @Test
+    public void testControllerException() throws Exception {
+        {
+            getRestTemplate().getForEntity("/exception", String.class);
+            Awaitility.await().until(reportedSpansSize(), IsEqual.equalTo(1));
+        }
+        List<MockSpan> mockSpans = TracingBeansConfiguration.mockTracer.finishedSpans();
+        Assert.assertEquals(1, mockSpans.size());
+        assertOnErrors(mockSpans);
+
+        MockSpan span = mockSpans.get(0);
+
+        Assert.assertEquals("GET", span.operationName());
+        Assert.assertEquals(6, span.tags().size());
+        Assert.assertEquals(Tags.SPAN_KIND_SERVER, span.tags().get(Tags.SPAN_KIND.getKey()));
+        Assert.assertEquals("GET", span.tags().get(Tags.HTTP_METHOD.getKey()));
+        Assert.assertEquals(getUrl("/exception"), span.tags().get(Tags.HTTP_URL.getKey()));
+        Assert.assertEquals(500, span.tags().get(Tags.HTTP_STATUS.getKey()));
+        Assert.assertNotNull(span.tags().get(Tags.COMPONENT.getKey()));
+        Assert.assertEquals(Boolean.TRUE, span.tags().get(Tags.ERROR.getKey()));
```

Comment:
Yes, I see this now. Thanks @pavolloffay. (I was previously looking for non-test code that sets the error tag)